examples/dotnet/dotnet-03-simple-chatbot/dotnet-03-simple-chatbot.csproj

```diff
@@ -60,4 +63,9 @@
         </PackageReference>
     </ItemGroup>
 
+    <ItemGroup>
+        <ProjectReference
+            Include="..\..\..\libraries\dotnet\WorkbenchConnector\WorkbenchConnector.csproj" />
```

Comment:
@tommasodotNET looks like the changes to the connector are just aesthetic, so we could keep using the nuget?